src/utils/outputs/download.ts

```diff
@@ -67,6 +72,33 @@ export const downloadTheme = async (
   const zip = new JSZip();
   zip.file(`${fileName}.json`, themeJsonString);
 
+  // Style dictionary
+
+  const sdFolder: string = "StyleDictionary";
+  zip.file(
+    `${sdFolder}/palette-colors.json`,
+    JSON.stringify(getSDColorPalette(allColors, luminanceSteps)),
+  );
+  zip.file(
+    `${sdFolder}/speaking-colors.json`,
+    JSON.stringify(getSDSpeakingColors(speakingNames, allColors)),
+  );
+  const token = getDBNonColorToken(theme);
+  const tokenProps = [
+    "spacing",
+    "sizing",
+    "border",
+    "elevation",
+    "transition",
+    "font",
+  ];
+  for (const prop of tokenProps) {
+    zip.file(
+      `${sdFolder}/${prop}s.json`,
+      JSON.stringify({ [prop]: token[prop] }),
+    );
+  }
```

Comment:
**issue**: Style Dictionary Tokens need a `value` property, to be recognised.

This code generates something like this: 

```json
{
  "border": {
    "height": {
      "3xs": "0.0625rem",
```

but we need 

```json
{
  "border": {
    "height": {
      "3xs": {
        "value": "0.0625rem",
      }
```